activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -276,7 +277,13 @@ def _active_record_fixture(fixture_set_name, *fixture_names)
           raise StandardError, "No fixture set named '#{fixture_set_name.inspect}'"
         end
       end
-      alias_method :fixture, :_active_record_fixture
+
+      # Generic fixture accessor for fixture names that may conflict.
```

Comment:
```suggestion
      # Generic fixture accessor for fixture names that may conflict with other methods.
```

The first time reading this made me think "conflict with what?" so maybe its a good idea to clarify